articles/azure-monitor/agents/azure-monitor-agent-supported-operating-systems.md

```diff
@@ -65,8 +65,8 @@ This article lists the operating systems that the [Azure Monitor Agent](./azure-
 | Red Hat Enterprise Linux Server 8.6+                        | ✓<sup>2</sup> |
 | Red Hat Enterprise Linux Server 8.0-8.5                     | ✓ |
 | Red Hat Enterprise Linux Server 7                           | ✓ |
-| Rocky Linux 9                                               | ✓ |
-| Rocky Linux 8                                               | ✓ |
+| Rocky Linux 9                                               | ✓<sup>2</sup> |
```

Comment:
@dylanbun rocky aarch64 was just added and isn't released yet: https://github.com/Azure/azure-linux-extensions/pull/2020, so let's not add this until released.

Reply:
@dylanbun rocky9 as well is not yet supported aarch64